R/ancova.R

```diff
@@ -28,19 +28,21 @@
 #' If no value for `visits` is provided then it will be set to
 #' `unique(data[[vars$visit]])`.
 #'
-#' In order to meet the formatting standards set by [analyse()] the results
-#' will be collapsed into a single list prefixed by the visit name e.g.:
-#' ```
-#' list(
-#'     visit_1_trt = list(est = ...),
-#'     visit_1_lsm_ref = list(est = ...),
-#'     visit_1_lsm_alt = list(est = ...),
-#'     visit_2_trt = list(est = ...),
-#'     visit_2_lsm_ref = list(est = ...),
-#'     visit_2_lsm_alt = list(est = ...),
-#'     ...
-#' )
-#' ```
+#' In order to meet the formatting standards set by [analyse()] the results will be collapsed
+#' into a single list suffixed by the visit name, e.g.:
+#'```
+#'list(
+#'    trt_visit_1 = list(est = ...),
+#'    lsm_ref_visit_1 = list(est = ...),
+#'    lsm_alt_visit_1 = list(est = ...),
+#'    trt_visit_2 = list(est = ...),
+#'    lsm_ref_visit_2 = list(est = ...),
+#'    lsm_alt_visit_2 = list(est = ...),
+#'    ...
+#')
+#'```
+#' Please note that "ref" refers to the first level of `vars$group` which not necessarily
+#' coincides with the control arm.
```

Comment:
Maybe just add a small note pointing out that "alt" refers to the 2nd level of the factor  and that "trt" refers to level 2 - level 1  i.e. standard factor / model contrasts